src/router.js

```diff
@@ -15,20 +18,19 @@ export default function(app) {
     },
     events: {
       loaded: function(state, actions) {
-        addEventListener("popstate", function() {
+        match()
```

Comment:
@SkaterDad By calling the local function match here, we trigger actions.router.match which calls emit("route"). So, the reason there's this match() here is so that events.route is triggered when the app loads.